spec/aria-reflection.html

```diff
@@ -0,0 +1,761 @@
+
+      // if your specification has a subtitle that goes below the main
+      // formal title, define it here
+      // subtitle   :  "an excellent document",
+
+      // if you wish the publication date to be other than the last modification, set this
+      // publishDate:  "2009-08-06",
+
+      // if the specification's copyright date is a range of years, specify
+      // the start date here:
+      // copyrightStart: "2005"
+
+      // if there is a previously published draft, uncomment this and set its YYYY-MM-DD date
+      // and its maturity status
+      // previousPublishDate:  "1977-03-15",
+      // previousMaturity:  "WD",
+
+      // if there a publicly available Editor's Draft, this is the link
+      // edDraftURI:           "http://berjon.com/",
+
+      // if this is a LCWD, uncomment and set the end of its review period
+      // lcEnd: "2009-08-05",
+
+      // editors, add as many as you like
+      // only "name" is required
+      editors: [{
+          name: "Alice Boxhall",
+          url: "http://google.com",
+          mailto: "aboxhall@google.com",
+          company: "Google",
+          companyURL: "http://google.com/"
+        },
+        {
+          name: "James Craig",
+          url: "http://apple.com",
+          mailto: "jcraig@apple.com",
+          company: "Apple",
+          companyURL: "http://apple.com/"
+        },
+        {
+          name: "Dominic Mazzoni",
+          url: "http://google.com",
+          mailto: "dmazzoni@google.com",
+          company: "Google",
+          companyURL: "http://google.com/"
+        },
+        {
+          name: "Alexander Surkov",
+          url: "http://mozilla.org/",
+          mailto: "surkov.alexander@gmail.com",
+          company: "Mozilla",
+          companyURL: "http://mozilla.org/"
+        },
+      ],
+      // name of the WG
+      //         wg:           "None",
+
+      // URI of the public WG page
+      //         wgURI:        "http://example.org/really-cool-wg",
+
+      // name (without the @w3c.org) of the public mailing to which comments are due
+      //          wgPublicList: "spec-writers-anonymous",
+
+      // URI of the patent status for this WG, for Rec-track documents
+      // !!!! IMPORTANT !!!!
+      // This is important for Rec-track documents, do not copy a patent URI from a random
+      // document unless you know what you're doing. If in doubt ask your friendly neighbourhood
+      // Team Contact.
+      //        wgPatentURI:  "",
+      // !!!! IMPORTANT !!!! MAKE THE ABOVE BLINK IN YOUR HEAD
+    };
+  </script>
+</head>
+
+<body>
+  <section id="abstract">
+    <p>
+      This specification describes the additions to existing specifications which will make it possible for web authors to programmatically
+      express Element semantics in two ways:
+      <ul>
+        <li>
+          <em>reflected</em> as DOM attributes on Elements</li>
+        <li>
+          <em>non-reflected</em>, via a ShadowRoot.</li>
```

Comment:
How about something like "providing default semantics for a custom element via a ShadowRoot (non-reflected)". I just feel like reflected vs non-reflected is more of an aspect, or a consequence, of the two places where you can set attributes - not the primary difference.

Reply:
I think the "default semantics" thing is a use case, but not necessarily the only one, so I don't want to state it that way. I'll just remove the reflected/non-reflected language.